Analysis/Core/include/Analysis/TrackSelection.h

```diff
@@ -0,0 +1,105 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+//
+// Class for track selection
+//
+
+#ifndef TrackSelection_H
+#define TrackSelection_H
+
+#include "Analysis/TrackSelectionTables.h"
+#include "Framework/AnalysisTask.h"
+
+class TrackSelection {
```

Comment:
I think this should become a ROOT class, so that we can stream it

Reply:
I think you still miss the ClassDef and the implementation macro in the cxx, or am I overlooking it?

Reply:
Indeed, I forgot that. It is added now with the latest commit.